### PR TITLE
[module] servo_tester update: automatic sweep option

### PR DESCRIPTION
**Pull request by dewagter**
Code from @hjkarssies

### CODE REVIEW

File: sw/airborne/modules/servo_tester/servo_tester.c [hunk 1]
```diff
@@ -80,5 +91,4 @@ void servo_tester_periodic()
 
   Bound(servo_test_val, SERVO_SERVO_TEST_MIN, SERVO_SERVO_TEST_MAX);
   Set_SERVO_TEST_Servo(servo_test_val);
-}
-
+}
```
Comment (gautierhattenberger) [inline sw/airborne/modules/servo_tester/servo_tester.c:94]: run format script ?

File: conf/modules/servo_tester.xml [hunk 2]
```diff
@@ -13,13 +13,17 @@
       amplitude. Tip: Open up the servo and measure the voltage on the potmeter with
       a high speed recorder device (like a saleae Logic analyzer with analog inputs). This 
       will let you model the response of the servo.
+	  
+      If you set the boolean "IS_SERVO" to False, the program wil run from minimum to maximum
```
Comment (gautierhattenberger) [inline conf/modules/servo_tester.xml:17]: typo: will

File: sw/airborne/modules/servo_tester/servo_tester.c (hunk 1 above)
Comment (dewagter) [reply sw/airborne/modules/servo_tester/servo_tester.c:96]: fixed

File: conf/modules/servo_tester.xml (hunk 2 above)
Comment (dewagter) [reply conf/modules/servo_tester.xml]: fixed

**Pull request merged**